db/migrate/20241205142318_add_more_native_columns_to_contacts.rb

```diff
@@ -0,0 +1,23 @@
+class AddMoreNativeColumnsToContacts < ActiveRecord::Migration[8.0]
+  def change
+    change_table :contacts do |t|
+      t.string :status, null: false, default: 'active'
+      t.string :language_code
```

Comment:
All of these native fields probably should be indexed. Because the whole point in making them native fields is that so they can be queried